StripePaymentsUI/StripePaymentsUI/Source/Helpers/STPStringUtils.swift

```diff
@@ -66,4 +66,28 @@ import Foundation
         ).first
         return result != nil && (result?.numberOfRanges ?? 0) > 0
     }
+
+    static let slashFormattedExpirationDateRegex: NSRegularExpression? = {
+        return try? NSRegularExpression(pattern: #"\b(\d{2})[/\.](\d{2,4})\b"#)
```

Comment:
`[/\.]`
this means there is a `/` or a `.` ?  if so, seems like we should have a test for this?

Reply:
Eh, I think we should only do `/` for now, I can't actually find any cards where we'd use a `.`. It's just more specific than the old one (which allows for any 1-3 non-digit characters) in between